tests/Fixtures/Integration/misc/PHP8_1.test

```diff
@@ -0,0 +1,19 @@
+--TEST--
+PHP 8.1 test.
```

Comment:
I see this PR not really adding `fsync` support, but showing that it's already supported, right?
maybe we should treat this PR as beginning of integration tests for 8.1 and start filling it with other 8.1-related cases?